plugins/chat/assets/stylesheets/mobile/chat-header.scss

```diff
@@ -0,0 +1,38 @@
+.chat-enabled.has-full-page-chat {
+  .d-header-icons .search-dropdown,
+  .d-header-icons .hamburger-dropdown {
+    display: none;
+  }
+
+  .home-logo-wrapper-outlet {
+    width: 100%;
+  }
+
+  .chat-header {
+    width: 100%;
+    height: 55px;
+    display: flex;
+    align-items: center;
+
+    a {
+      padding: 10px;
```

Comment:
padding in rems please + why not add it to the classes `back-to-forum` and `chat-heading` already used below instead of the a-tag